src/test/java/htsjdk/samtools/reference/AbstractIndexedFastaSequenceFileTest.java

```diff
@@ -353,4 +354,16 @@ public void testBadInputForIndexedFastaSequenceFile() throws Exception {
     public void testBadInputForBlockCompressedIndexedFastaSequenceFile() throws Exception {
         new BlockCompressedIndexedFastaSequenceFile(SEQUENCE_FILE.toPath());
     }
+
+    @Test
+    public void testCanCreateBlockCompressedIndexedWithSpecifiedGZIAndDict() throws IOException {
+        final Path moved = Files.createTempFile("moved", ".fasta.gz");
+        Files.copy(SEQUENCE_FILE_BGZ.toPath(), moved, StandardCopyOption.REPLACE_EXISTING);
+        IOUtil.deleteOnExit(moved);
+        try( ReferenceSequenceFile withNoAdacentIndex = new BlockCompressedIndexedFastaSequenceFile(moved, new FastaSequenceIndex(SEQUENCE_FILE_INDEX), GZIIndex.loadIndex(SEQUENCE_FILE_GZI.toPath()));
+             ReferenceSequenceFile withFilesAdjacent = new BlockCompressedIndexedFastaSequenceFile(SEQUENCE_FILE_BGZ.toPath())){
```

Comment:
Indent looks wrong on this line but it's hard to tell for sure in the PR. I would expect it to line up with the text after the `(` on the previous line.

Reply:
It's lined up with the beginning of the other ReferenceSequenceFile, so it's 1 space after the `(` I think it's what you expect.  Auto format doesn't change anything.